packages/financial-contracts/subgraph.template.yaml

```diff
@@ -31,24 +31,6 @@ dataSources:
         - event: NewContractRegistered(indexed address,indexed address,address[])
           handler: handleNewContractRegistered
 templates:
-  - name: CollateralERC20
-    kind: ethereum/contract
-    network: {{network}}
-    source:
-      abi: ERC20
-    mapping:
-      kind: ethereum/events
-      apiVersion: 0.0.4
-      language: wasm/assemblyscript
-      file: ./src/index.ts
-      entities:
-        - FinancialContract
-      abis:
-        - name: ERC20
-          file: ./abis/ERC20.json
-      eventHandlers:
-        - event: Transfer(indexed address,indexed address,uint256)
-          handler: handleCollateralTransfer
```

Comment:
Not tracking this speeds up the indexing process a lot because we're no longer indexing all Transfer events for each collateral currency